sql/hive/src/test/resources/golden/Test UDTF.close in Lateral Views-0-ac5c96224a534f07b49462ad76620678

```diff
@@ -0,0 +1,2 @@
+97	500
+97	500
```

Comment:
Lets not put the test jar in `resources/data/files/`, as that is a directory that we copied from hive, and might replace in the future.